build/PackageDiffIgnore.xml

```diff
@@ -3071,6 +3071,18 @@
       <Member fullName="System.Void Windows.System.UserProfile.UserProfilePersonalizationSettings..ctor()"
             reason="Parameter-less ctor does not exist in UWP"/>
 			<Member
+              fullName="System.Void Windows.Devices.Geolocation.Geolocator.OnStatusChanged(System.String provider, Android.Locations.Availability status, Android.OS.Bundle extras)"
```

Comment:
If we're making a breaking change, let's move those functions in a nested class instead.

Reply:
I don't remember exactly why I created this ignore; but it seems that this is Android method (with Android parameters).
What do you mean by "nested class" in this context?

Reply:
What do you mean by "nested class" in this context?

Reply:
Create a [nested class](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/nested-types) for all methods are implemented for `ILocationListener`.

Reply:
Your comment is valid for current implementation - as current implementation makes Geolocator inherits from ILocationListener. But I don't touch current implementation, I'm only adding correct GetGeopositionAsync(TimeSpan maximumAge, TimeSpan timeout) implementation.

Reply:
I'm closing this, as this PR doesn't modify DiffIgnore now.